### DISCUSSION TITLE
Should we have discussions?

### DISCUSSION THREAD

**Discussion by lazka** [msys2/MINGW-packages#7413] [Ideas]
This is now available for all repos it seems. Should we give it a try?

**Comment by Alexpux** [comment]
Yes I think it is good idea

**Comment by pbek** [comment]
Nice!

**Comment by eine** [comment]
It seems that discussions are numbered, just as Issues and PRs. Is it possible to convert issues into discussions? Replying to specific messages is a nice feature. But I'm afraid that maybe discussions cannot be labeled/milestoned.

**Reply by lazka** [reply]
> Is it possible to convert issues into discussions?

yes

**Comment by revelator** [comment]
To much information 💯  nah its nice :) though ill have to get used to the options around here, still banging my head around splitting pull requests. 

Happy new year to all involved though 👍
